config/auth.php

```diff
@@ -112,4 +112,5 @@
         ],
     ],
 
+    'iar' => env('APP_REGISTER', false),
```

Comment:
For ease of understanding the code base an control flows, I would love for us to be a bit more verbose with our configuration keys, it will read more fluently and avoid having to maintain a map of short names to actual meaning

```suggestion
    'in_app_registration' => env('APP_REGISTER', false),
```